CMakeLists.txt

```diff
@@ -4,11 +4,17 @@ if(POLICY CMP0092)
     cmake_policy(SET CMP0092 NEW)
 endif()
 
-# The value of this variable should be set prior to the first project() command invocation.
-# See: https://cmake.org/cmake/help/latest/variable/CMAKE_OSX_DEPLOYMENT_TARGET.html
-if(NOT CMAKE_OSX_DEPLOYMENT_TARGET)
-    set(CMAKE_OSX_DEPLOYMENT_TARGET "10.13" CACHE STRING "Minimum OS X version to target for deployment" FORCE)
-endif()
+function(tr_set_apple_deployment_targets OSX_DEPLOYMENT_TARGET)
```

Comment:
Not sure why it's plural.
```suggestion
function(tr_set_apple_deployment_target OSX_DEPLOYMENT_TARGET)
```

Reply:
Hopefully we'll have iOS support in the future?

Reply:
I reverted most changes.